thunder/benchmarks/__init__.py

```diff
@@ -1221,12 +1217,14 @@ def args(cls) -> tuple[BenchmarkArg, ...]:
 
     def __init__(
         self,
-        config: str | LitGPTConfig,
+        config: str,
```

Comment:
nit: since `LitGPTConfig` is still a valid thing to pass, maybe `str | Any`, or even `str | "LitGPTConfig"`?

Reply:
Since it's a nit I'll keep it as is. We don't use type checkers anyway.

Reply:
I'm pretty sure the next person will want to have their type checker enabled, but well.